задание 1

andreyzavrichko opened the pull request.

Commit:
задание 1

mkouznetsov-git commented on src/test/java/praktikum/BurgerTests.java:
```diff
@@ -0,0 +1,75 @@
+    Ingredient ingredient;
+
+    @Test
+    public void getPriceBurgerTest() {
```
Надо бы не каждую функцию отдельный тест сделать. move, add, remove.

Commit:
добавил тесты
- verifyMoveIngredient
- verifyDeleteIngredient
- verifyAddIngredient
новый отчет
